### DISCUSSION TITLE
Buttons on notification

### DISCUSSION THREAD

**Discussion by Bindslev** [thudugala/Plugin.LocalNotification#372] [Q&A] [answered]
Hello
Is it possible to add action buttons on notifications so that the user can click those buttons and it is registered in the app which button was clicked? 
An example: [Notification on Android Studio](https://4.bp.blogspot.com/-8ivBScnqJ54/XG2N84DHysI/AAAAAAAAIFA/3DwD2FKjWw8B_5YnNWX-EwgbQQDEHS_bQCLcBGAs/s1600/Screenshot%2B%2528103%2529.png)
This example showcases a notification where two action buttons are present, "like" and "dislike". Usually you'd click these buttons and the notification would then disappear. It would be really awesome to have something similar to this, but I have yet to find it.
In the video by Gerald, he mentions this is indeed possible, so I'm just looking for an example on how this is done.
Thank you

**Comment by thudugala** [comment, marked as answer]
@Bindslev 

https://github.com/thudugala/Plugin.LocalNotification/wiki/Notification-with-Action
